sonic-slave-bookworm/Dockerfile.j2

```diff
@@ -417,6 +417,9 @@ RUN apt-get update && apt-get install -y \
 # For audisp-tacplus
         libauparse-dev \
         auditd \
+# For nvidia
```

Comment:
Is it possible to only have them installed on our platform?

Reply:
The slave docker is not platform-specific. In this "apt-get install" we install the packages for all the platforms.